src/Input/Select/Select.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import Select, { Props as SelectProps } from './Select';
+import { WarningOutlineIcon } from '../../General/Icon/components';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'Input/Select',
+  component: Select,
+  argTypes: {
+    status: {
+      control: null,
+    },
+    error: {
+      control: { type: 'boolean' },
+    },
+  },
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+} as Meta;
+
+const Options = [
+  <Select.Option key={1} value="accountant">
+    Accountant
+  </Select.Option>,
+  <Select.Option key={2} value="business development">
+    Business Development
+  </Select.Option>,
+  <Select.Option key={3} value="software engineer">
+    Software Engineer
+  </Select.Option>,
+];
+
+const Template: Story<SelectProps> = args => (
+  <div style={{ height: '200px' }}>
+    <Select
+      label="Jobs"
+      {...args}
+      // set undefined to prevent these properties are shown in code sample
+      onBlur={undefined}
+      onFocus={undefined}
+      onChange={undefined}
+      onInputChange={undefined}
```

Comment:
I'm afraid that those properties might distract the user's attention so use this approach to hide them.
![Screen Shot 2020-12-03 at 10 29 41 PM](https://user-images.githubusercontent.com/13249748/101043060-d47abc00-35b8-11eb-819e-40e5e5a2b316.png)

Reply:
I think we can accept that for the time being. Hiding the problem won't make it go away. We can investigate this more in the future or hope that StoryBook pushes an update that deals with this.

Reply:
Hmm, I think I will keep undefined there because those functions are not listed in the control table. But yeah, we should investigate this more in the future as well. 🙏